File: launchdarkly/ruby/src/GetCustomWorkflowExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::WorkflowsApi.new.get_custom_workflow(
        "projectKey_string", # project_key
        "featureFlagKey_string", # feature_flag_key
        "environmentKey_string", # environment_key
        "workflowId_string", # workflow_id
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling WorkflowsApi#get_custom_workflow: #{e}"
end

File: launchdarkly/python/src/GetCustomWorkflowExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.WorkflowsApi(api_client).get_custom_workflow(
            project_key="projectKey_string",
            feature_flag_key="featureFlagKey_string",
            environment_key="environmentKey_string",
            workflow_id="workflowId_string",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling WorkflowsApi#get_custom_workflow: %s\n" % e)
